### ISSUE TITLE
[TLP]: 

### ISSUE THREAD

**Issue by iotahouse** [soonaverse/soonaverse-dao#829]
### Email

iotahouse0@gmail.com

### Company Name

IOTA House

### Your Position

CEO

### Name

IOTA House

### Have you already done a public sale?

No

**Comment by iotahouse** [comment]
Dear

Can you require/accept us the kyc badge to buy it afterwards ? :)

We submitted the token creation  and would like to launch as soon as possible hopefully. :)

Kind regards
IOTA House

**Comment by adamunchained** [comment]
KYC badge must happen prior the token sale. It can be very quick tbh. You still got plenty of time.

@shonuffy fyi

**Comment by shonuffy** [comment]
Did you request the applicant award using IOTAHOUSE? I'm not seeing your request.

**Comment by iotahouse** [comment]
I did that with the first "tlp kyc" but now it should be okay since i did the kyc applicant award.

you can confirm if you received now

Thanks

**Comment by shonuffy** [comment]
Confirmed and approved, please purchase the KYC NFT and we will forward your information to our vendor.

**Comment by iotahouse** [comment]
I purchased the KYC NFT. Can you forward our information to the vendor today so i can send the vendor the needed information.


Thank you
IOTA House

**Comment by shonuffy** [comment]
Please be advised that we are not approving token listings until after the launch of Shimmer

**Comment by iotahouse** [comment]
Allright thanks, can you then make sure we're listed as soon as possible after the launch of Shimmer ?  :)
Btw is there a date planned for the Shimmer launch or ? :)

Thank you
IOTA House

**Comment by iotahouse** [comment]
Do we need to receive something from the vendor and send something or is everything now fine because we haven't received anything ?

kind regards
IOTA House

**Comment by iotahouse** [comment]
Okay, i am looking forward to get a response and launch as soon as possible our tokens on soonaverse.


IOTA House

**Comment by shonuffy** [comment]
Congratulations, you will be issued the KYC applicant award on September 26. This will give you time to set up your token and launch once Shimmer goes live on the 28th. 

Please go to wen2.soonaverse.com to test out the system to learn how to configure your token set up and launch strategy. Any mistakes you make on the mainnet CANNOT be reversed. Again, we highly recommend you go through this test run before you set up your token for the launch. 

Also, based on the updated guidance of our legal advisory, Native tokens cannot be purchased by users in the United States and Canada without a legal opinion that you are not a security. 

Thank you for going through our set up process and good luck

**Comment by shonuffy** [comment]
@iotahouse
